www/docs/en/dev/guide/platforms/ios/index.md

```diff
@@ -49,6 +49,18 @@ The table below outlines Cordova-iOS's prerequisites by version. Additionally, X
         </tr>
     </thead>
     <tbody>
+        <tr>
+            <td>8.x</td>
+            <td>13.0</td>
+            <td>
+                <ul>
+                    <li>Cocoapods: >=1.16.0</li>
+                    <li>ios-deploy: >=1.12.2</li>
+                    <li>Node: >=16.13.0</li>
```

Comment:
I think our intention is to target NodeJS 18? But that would put us out of sync with the rest of the Cordova tooling... but I also don't want to delay the iOS release until we're ready to depend on 20 for everything 😕